dirty_equals/_boolean.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, Optional, TypeVar
+
+from ._base import DirtyEquals
+
+B = TypeVar('B')
+
+
+class IsFalseLike(DirtyEquals[B]):
+    """
+    Check if the value is False like, and matches the given conditions.
+    """
+
+    def __init__(
+        self,
+        *,
+        allow_strings: Optional[bool] = None,
+    ):
+        """
+        Example of basic usage:
+
+        ```py title="IsFalseLike"
+        from dirty_equals import IsFalseLike
+
+        assert False == IsFalseLike
+        assert 0 == IsFalseLike
+        assert '0' == IsFalseLike(allow_strings=True)
```

Comment:
put the `assert 'false' == IsFalseLike(allow_strings=True)` case before these.

You should also add `assert 'foobar' != IsFalseLike(allow_strings=True)` to show that other values don't raise an error.